pkg/controller/machine/kubeconfig.go

```diff
@@ -42,6 +42,8 @@ const (
 	tokenSecretKey           string            = "token-secret"
 	expirationKey            string            = "expiration"
 	tokenFormatter           string            = "%s.%s"
+	// Keep this short, userdata is limited
+	contextIdentifier string = "c"
```

Comment:
Pretty short. ;)

Reply:
yeah, the problem we have is that the userdata size is limited, so I wanted to keep it short :P